src/ui/actions/comments.ts

```diff
@@ -47,7 +47,7 @@ export function updatePendingCommentContent(content: string): UpdatePendingComme
 export function createComment(
   time: number,
   point: string,
-  user: User,
+  user: User, // TODO: user is no longer required and should be removed
```

Comment:
Chatted with Josh, `user` is no longer required so we should safely be able to refactor this and get rid of it in the future.